### ISSUE TITLE
Confusing error if your password is incorrect

### ISSUE THREAD

**Issue by statico** [aws-azure-login/aws-azure-login#3]
This tool is awesome, but it took me a long time to figure out I was entering the wrong password.

If you enter a wrong password you get:

```
$ aws-azure-login
? Username: me@example.com
? Password: **************
We don't recognize this user ID or password
```

If you enter a previously-used but no longer valid password:

```
$ aws-azure-login
? Username: me@example.com
? Password: **************
Did you change your password recently?
```

I had no idea in the latter case that the program had failed. `aws s3 ls` would just say `Unable to locate credentials. You can configure credentials by running "aws configure".`

It would be super helpful if the program would say explicitly "Authentication failed" as well as exit with a non-zero exit code.

Thanks!

**Comment by dtjohnson** [comment]
Sure. Easy enough. The app was simply returning the error message exactly as it was in Azure. I just published v0.3.1 that prefixes those errors with "Login failed" to make it clear. I also added a non-zero exit code.

**Comment by statico** [comment]
Thank you! :D
